internal/cephfs/volumeoptions.go

```diff
@@ -475,6 +475,14 @@ func newSnapshotOptionsFromID(ctx context.Context, snapID string, cr *util.Crede
 	sid.FsSnapshotName = imageAttributes.ImageName
 	sid.FsSubvolName = imageAttributes.SourceName
 
+	subvolInfo, err := getSubVolumeInfo(ctx, &volOptions, cr, volumeID(sid.FsSubvolName))
+	if err == nil {
+		volOptions.Features = subvolInfo.Features
+	}
+	if !errors.Is(err, ErrInvalidCommand) {
```

Comment:
we dont need this check. snapshot creation is not supported if the `subvolume info` command is not supported just return errror

Reply:
Okay, This will not break any backward compatibility. Right?

Reply:
Yes it should work